app/views/comments/_comment.html.erb

```diff
@@ -5,7 +5,14 @@
     <br>

```

Comment:
If the current user is the author of the comment, they will not see their own name on the comment?